semeio/jobs/rft/trajectory.py

```diff
@@ -25,14 +35,31 @@ def __init__(self, utm_x, utm_y, measured_depth, true_vertical_depth, zone=None)
         self.measured_depth = measured_depth
         self.true_vertical_depth = true_vertical_depth
         self.zone = zone
-        self.grid_ijk = None
+        self.grid_ijk = None  # tuple
         self.pressure = None
+        self.swat = None
+        self.sgas = None
+        self.soil = None
         self.valid_zone = False
 
     def set_ijk(self, point):
```

Comment:
Slightly outside the scope, but should this be a `property` with validation on the `setter`? Can be a separate issue  as you are just updating the doc string.

Reply:
I think I will leave this for later.